.changeset/angry-kiwis-raise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@telegraph/button": minor
+---
+
+Modify button default behavior to NOT submit form
```

Comment:
fyi @andy-knock we typically use conventional commits for all commit messages and release notes